src/root/components/AuthenticatedLayout/AuthenticatedLayout.scss

```diff
@@ -1,3 +1,6 @@
+// AuthenticatedLayout Sizes
+$menu-width: 70px;
```

Comment:
would always start with the component name:

`$authenticatedlayout-menu-width`